src/terminal/parser/InputStateMachineEngine.cpp

```diff
@@ -846,7 +847,8 @@ DWORD InputStateMachineEngine::_GetModifier(const size_t modifierParam) noexcept
 bool InputStateMachineEngine::_UpdateSGRMouseButtonState(const VTID id,
                                                          const size_t sgrEncoding,
                                                          DWORD& buttonState,
-                                                         DWORD& eventFlags) noexcept
+                                                         DWORD& eventFlags,
+                                                         COORD uiPos)

```

Comment:
We cannot call `_lastMouseClickPos = uiPos` or `_lastMouseClickTime = currentTime` if this function is declared `noexcept`, so I've removed it unless someone has a better suggestion!